style.css

```diff
@@ -3,385 +3,1016 @@
-}
-
-.info-form {
-  justify-content: center;
-  text-align: center;
-  width: 300px;
-  display: flex;
-  flex-direction: column;
-  align-content: center;
-}
-
-.h003 {
-  font-family: "Poppins", sans-serif;
-  font-style: normal;
-  font-weight: 400;
-  font-size: 20px;
-  line-height: 28px;
-
-  /* or 140% */
-
-  text-align: center;
-
-  /* Primary/B50 */
-
-  color: #ebebff;
-}
-
-.input {
-  background: #fff;
-
-  /* Base/100 */
-  font-size: 16px;
-  border: 1px solid #cfd8dc;
-  border-radius: 8px;
-  height: 3rem;
-  margin-top: 1rem;
-  width: 20rem;
-  padding-left: 1rem;
-}
-
-.mssg {
-  background: #fff;
-
-  /* Base/100 */
-  padding-left: 1rem;
-  border: 1px solid #cfd8dc;
-  border-radius: 8px;
-  height: 8rem;
-  font-size: 16px;
-  margin-top: 1rem;
-  width: 20rem;
-}
-
-.formy-btn {
-  margin-top: 1rem;
-  margin-left: -9rem;
-  padding: 1rem 2rem 1rem;
-}
-
-form {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-}
-
-.formy-btn:hover {
-  background: skyblue;
-  color: white;
+@media only screen and (max-width: 768px) {
+  body {
+    background-image: url("./image/body.png");
+    background-size: 80%;
+    background-position: center;
+    background-position-y: 4%;
+    background-repeat: no-repeat;
+    margin-bottom: 2rem;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .swapcard3 {
+    display: none;
+  }
+
+  .main {
+    display: grid;
+    grid-template-columns: auto;
+    padding-left: 2rem;
+  }
+
+  header {
+    width: 70%;
+    padding: 12px;
+    padding-top: 3rem;
+    margin-bottom: 5rem;
+    grid-template-columns: auto auto;
+    gap: 100%;
+    display: grid;
+  }
+
+  a > .sp1 {
+    flex-grow: 70%;
+    padding-left: 1%;
+    width: 75px;
+    height: 20px;
+    font-weight: 700;
+    font-size: 30px;
+    line-height: 20px;
+    color: #6070ff;
+  }
+
+  .sp2 {
+    padding-right: 1%;
+  }
+
+  .paragraph {
+    widows: 70%;
+    height: auto;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .text {
+    padding-left: 15px;
+    width: 80%;
+    height: 150px;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    color: #172b4d;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .text2 {
+    margin-top: 1%;
+    margin-bottom: 5%;
+    padding-left: 15px;
+    width: 80%;
+    height: 146px;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  .card001 {
+    display: grid;
+    grid-template-columns: auto;
+  }
+
+  .a {
+    display: flex;
+    padding-left: 15px;
+    align-self: initial;
+    font-weight: 500;
+    font-size: 16px;
+    text-decoration: none;
+    line-height: 24px;
+    color: #7f8cff;
+  }
+
+  .social-media {
+    display: flex;
+    widows: 300px;
+    justify-content: space-between;
+    max-width: 150px;
+    margin-top: 3rem;
+    padding-inline-start: 15px;
+  }
```

Comment:
- [ ] As it was in the project requirement that we should stick to the Figma template as much as we can, I noticed there are still some errors in the header design of your `desktop` version. The `let's connect` and the `social media icons` are not well `aligned` with the header description.

Your design  👇 

![isma1](https://user-images.githubusercontent.com/78634841/206787878-bfc0b062-3aff-4ff9-802c-5b64be0b19af.png)

 Figma design 👇 

![isma real](https://user-images.githubusercontent.com/78634841/206787368-a585bf32-5c3b-4eb3-93c8-f99a7b24834f.png)